### ISSUE TITLE
Add arn to the IoT role alias

### ISSUE THREAD

**Issue by jhosteny** [terraform-providers/terraform-provider-aws#8811]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I originally implemented this resource, but forgot to include the ARN. Now that I am attempting to use it more completely, I noticed that the iot:AssumeRoleWithCertificate needs a fully qualified ARN for the role alias (i.e., the region and account id must be specified) for authentication to work properly.

### New or Affected Resource(s)

* aws_iot_role_alias

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #2666

**Comment by bflad** [comment]
The new `arn` attribute for the `aws_iot_role_alias` resource has been merged and will release with version 2.13.0 of the Terraform AWS Provider, likely later this week. 👍

**Comment by bflad** [comment]
This has been released in [version 2.13.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2130-may-31-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
